src/transformers/default-schema-transformer.ts

```diff
@@ -276,3 +279,31 @@ function handleQuestionsWithObsComments(sectionQuestions: Array<FormField>): Arr
 
   return augmentedQuestions;
 }
+
+function handleDiagnosis(question: FormField) {
```

Comment:
@CynthiaKamau can you also configure the field to be search-able?

```ts
question.questionOptions.isSearchable = true;

```

Reply:
Done